src/game/game.cpp

```diff
@@ -935,10 +935,12 @@ namespace game
                     int millis = lastmillis-d->lastres[WR_BURN];
                     size_t seed = size_t(d) + (millis/50);
                     float pc = 1, amt = (millis%50)/50.0f, intensity = 0.75f+(detrnd(seed, 25)*(1-amt) + detrnd(seed + 1, 25)*amt)/100.f;
-                    if(burntime-millis < burndelay) pc *= float(burntime-millis)/float(burndelay);
+                    // When playing on servers with an older version, burndelay can be 0, thus add a check to prevent division by 0
```

Comment:
The comment is very long, and this is not a check you just enforce a lower boundary.

```suggestion
                    // avoid division by zero by enforcing new lower boundary
```